src/interface/guide/index.tsx

```diff
@@ -308,6 +308,11 @@ export const GoodMark = () => <i className="glyphicon glyphicon-ok good-mark" />
 export const OkMark = () => <i className="glyphicon glyphicon-asterisk ok-mark" />;
 export const BadMark = () => <i className="glyphicon glyphicon-remove bad-mark" />;
 
+export const PerfectColor = '#2090c0';
+export const GoodColor = '#4ec04e';
+export const OkColor = '#ffc84a';
+export const BadColor = '#ac1f39';
```

Comment:
These are copied from the same colors in CSS - is it possible for them to be accessible from TS and CSS without copying them?

Reply:
This seems like a possibly good case for CSS variables.

```tsx
export const PerfectColor = getComputedStyle(document.documentElement).getPropertyValue('--guide-perfect-color');
```

```scss
$perfect-perf-color: #2090c0;

:root {
  --guide-perfect-color: #{perfect-perf-color};
]
```

Reply:
Seems to work, thanks!